ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-bgp.md

```diff
@@ -1039,6 +1047,10 @@
             replace_as: <bool>
           session_tracker: <str>
 
+          # To apply a profile, it must be created under "management-security".
+          shared_secret_profile_name: <str>
+          hash_algorithm: <str; "aes-128-cmac-96" | "hmac-sha-256" | "hmac-sha1-96">
```

Comment:
It is not clear that this hash_algorithm is related to shared_secret stuff. I suggest creating a dict with the two related keys.

Reply:
Fixed.